shared/chat/dumb.js

```diff
@@ -306,75 +327,71 @@ const inboxParentProps = {
 }
 
 const conversationsList = {
-  component: ConversationListContainer,
+  component: Inbox,
   mocks: {
     'Normal': {
-      ...commonConversationsProps,
+      ...commonConversationsProps({}),
       parentProps: inboxParentProps,
     },
     'Selected Normal': {
-      ...commonConversationsProps,
+      ...commonConversationsProps({selected: 'convo1'}),
       parentProps: inboxParentProps,
-      selectedConversation: 'convo1',
     },
     'SelectedMuted': {
-      ...commonConversationsProps,
+      ...commonConversationsProps({selected: 'convo3'}),
       parentProps: inboxParentProps,
-      selectedConversation: 'convo3',
     },
     'Empty': {
       ...emptyConversationsProps,
       parentProps: inboxParentProps,
     },
     'PartRekey': {
-      ...rekeyConvo(false),
+      ...rekeyConvo('convo3', false),
       parentProps: inboxParentProps,
-      selectedConversation: 'convo3',
     },
     'PartRekeySelected': {
-      ...rekeyConvo(false),
+      ...rekeyConvo('convo1', false),
       parentProps: inboxParentProps,
-      selectedConversation: 'convo1',
     },
     'YouRekey': {
-      ...rekeyConvo(true),
+      ...rekeyConvo('convo3', true),
       parentProps: inboxParentProps,
-      selectedConversation: 'convo3',
     },
     'YouRekeySelected': {
-      ...rekeyConvo(true),
+      ...rekeyConvo('convo1', true),
       parentProps: inboxParentProps,
-      selectedConversation: 'convo1',
     },
     'LongTop': {
-      ...commonConversationsProps,
+      ...commonConversationsProps({
+        inbox: List([
+          new InboxStateRecord({
+            conversationIDKey: 'convo1',
+            info: null,
+            muted: false,
+            participants: List(['one', 'two', 'three', 'four', 'five', 'six', 'seven', 'eight', 'nine', 'ten']),
+            snippet: 'look up!',
+            time: now,
+            unreadCount: 3,
+          }),
+        ]),
+      }),
       parentProps: inboxParentProps,
-      inbox: List([
-        new InboxStateRecord({
-          conversationIDKey: 'convo1',
-          info: null,
-          muted: false,
-          participants: List(['one', 'two', 'three', 'four', 'five', 'six', 'seven', 'eight', 'nine', 'ten']),
-          snippet: 'look up!',
-          time: now,
-          unreadCount: 3,
-        }),
-      ]),
     },
     'LongBottom': {
-      ...commonConversationsProps,
+      ...commonConversationsProps({
+        inbox: List([
+          new InboxStateRecord({
+            conversationIDKey: 'convo1',
+            info: null,
+            muted: false,
+            participants: List(['look down!']),
```

Comment:
it is, the point of this is to get a very long snippet. the row above is the opposite, the snippet says 'look up!' and it has a very long participant list